### DISCUSSION TITLE
There are two directories in the centos7 environment. In fact, the first directory is cached.

### DISCUSSION THREAD

**Discussion by baerwang** [community/community#74615] [Actions]
### Select Topic Area

Bug

### Body

https://github.com/baerwang/pika/actions/runs/6796950283/job/18480177103?pr=4

**Comment by gitstua** [comment]
Hi @baerwang, if you have feedback about the runner image then you can raise this in the [Actions Runner Images repo](https://github.com/actions/runner-images#how-to-interact-with-the-repo)

Hope this helps!
